kube_proxy/MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include requirements.in
```

Comment:
can you also include `README.md`? Depending on setuptools version the source install (used by tox) might fail otherwise.